src/redux/modules/posts.js

```diff
@@ -82,6 +122,11 @@ const byId = (state = initialState.byId, action) => {
     switch(action.type){
         case types.FETCH_ALL_POSTS:
             return action.posts;
+        case types.FETCH_POST:
```

Comment:
While for the posts when getting post list, they don't have content loaded because for the list, it doesn't need it.
When fetch the details of one post, we need to load its content for sure, and we will use this more detailed post to overwrite the less detailed one in post list.